index.js

```diff
@@ -13,11 +13,17 @@ program
   .description(pkg.description);
 
 program
-  .command('top')
+  .command('top [articleCount]')
   .description('List Linda Ikeji Top Stories')
-  .option('-n, --number <int>", "specify number of stories')
-  .action(function(options){
-    var count = isNaN(parseInt(options) || parseInt(options) == 0) ? 20 : options;
+  .action(function(articleCount) {
```

Comment:
Because it was not being used (at least not in the format given)

> On 1 Jul 2016, at 3:28 AM, Prosper Otemuyiwa <notifications@github.com> wrote:
> 
> In index.js <https://github.com/unicodeveloper/lindaikeji-cli/pull/7#discussion_r69239938>:
> 
> >    .description('List Linda Ikeji Top Stories')
> > -  .option('-n, --number <int>", "specify number of stories')
> > -  .action(function(options){
> > -    var count = isNaN(parseInt(options) || parseInt(options) == 0) ? 20 : options;
> > +  .action(function(articleCount) {
> Any reason why the option method for specifying number of stories was removed?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/unicodeveloper/lindaikeji-cli/pull/7/files/3488b5bfdeb958cc84a1407f5e12a02b40a1cb68#r69239938>, or mute the thread <https://github.com/notifications/unsubscribe/ABROfA5N9fk8bwUg3kj_a32R1g0rSQjLks5qRHs1gaJpZM4JCrq2>.
>